app/src/main/java/com/example/snookerapi/adapters/PlayerViewHolder.kt

```diff
@@ -0,0 +1,31 @@
+package com.example.snookerapi.adapters
+
+import android.annotation.SuppressLint
+import androidx.recyclerview.widget.RecyclerView
+import coil.load
+import com.example.snookerapi.model.Player
+import com.example.snookerapi.R
+import com.example.snookerapi.databinding.ItemPlayerBinding
+
+class PlayerViewHolder(
+    private val binding: ItemPlayerBinding,
+    private val onPlayerClicked: (Player) -> Unit,
+) : RecyclerView.ViewHolder(binding.root) {
+
+    @SuppressLint("SetTextI18n")
+    fun bind(item: Player) {
+        with(binding) {
+            if (item.photoUrl != "") {
+                playerPhoto.load(item.photoUrl)
+            } else {
+                playerPhoto.load(R.drawable.no_photo)
+            }
+
+            playerFullName.text = item.firstName + " " + item.lastName
```

Comment:
делать соединение строк лучше как предлагает котлин
```suggestion
            playerFullName.text = "${item.firstName} ${item.lastName}"
```